### PR TITLE
Create test pull request

### PR DESCRIPTION
**Pull request by johannaut**
:-)

### CODE REVIEW

**Review by AlenaNaima — CHANGES_REQUESTED** [review]
Sorry no, this is not helpful

File: Examples/Goals.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Get rid of both of your instruments.
+* Play a consonance for at least 4 seconds.
+* Eat a sandwich.
```
Comment (AlenaNaima) [inline Examples/Goals.md]: I am not sure if this goal is achievable

**Review by AlenaNaima — APPROVED** [review]
Approved with one comment

File: Examples/Goals.md (hunk 1 above)
Comment (johannaut) [reply Examples/Goals.md]: Oh yes, maybe I change this goal.

**Pull request merged**